extensions/data-workspace/src/common/constants.ts

```diff
@@ -14,7 +14,7 @@ export const SelectProjectFileActionName = localize('SelectProjectFileActionName
 export const AllProjectTypes = localize('AllProjectTypes', "All Project Types");
 export const ProviderNotFoundForProjectTypeError = (projectType: string): string => { return localize('UnknownProjectTypeError', "No provider was found for project type with id: '{0}'", projectType); };
 export const RestartConfirmation = localize('dataworkspace.restartConfirmation', "Azure Data Studio needs to be restarted for the project to be added to the workspace. Restart now?");
-export const ProjectsFailedToLoad = localize('dataworkspace.projectsFailedToLoad', "Some projects failed to load. To view more details, [open the developer console](command:workbench.action.toggleDevTools)");
+export const projectFailedToLoad = (project: string, error: string) => { return localize('projectFailedToLoad', "Project '{0}' failed to load.  To view more details, [open the developer console](command:workbench.action.toggleDevTools): {1}", project, error) };
```

Comment:
it looked a little weird to me having the error message show after the link to open the developer console. Can they be swapped so the link to open the developer console is at the end?

Reply:
Yup, absolutely!  I thought that error messages in that toast got truncated at some point, so my goal was to make sure that link to the dev console was before that point, but it seems I was wrong, and the user just need to expand it if the message is longer than one line (which everything will be, so hopefully they always expand):

<img width="656" alt="image" src="https://user-images.githubusercontent.com/1609827/233435706-91b31689-fb8c-4df8-87d6-181017b0bb8f.png">

Reply:
Uhhh, let's maybe leave it for now.  It shouldn't contain any stacktraces, and we don't have any absurdly long errors that we know of in the Projects library.  It helps (compactness, but not readability) that the toast strips out any newlines, too.

Here's what it looks like now:

<img width="570" alt="image" src="https://user-images.githubusercontent.com/1609827/233464345-573c83c1-c299-4b05-a796-655f58a96c9e.png">